app/actions/views/channel.js

```diff
@@ -554,6 +569,14 @@ export function increasePostVisibility(channelId, focusedPostId) {
         if (posts) {
             hasMorePost = posts.order.length >= pageSize;
 
+            actions.push({
+                type: ViewTypes.INCREASE_POST_COUNT,
+                data: {
+                    channelId,
+                    count: posts.order.length,
```

Comment:
In `loadPostsIfNecessaryWithRetry`, we're combining posts before getting this count, but that's not happening here. Could that cause issues?

Reply:
actually I think it would be better if we combined them here, I'll make the change